src/main/java/org/jenkinsci/plugins/envinject/EnvInjectPluginConfiguration.java

```diff
@@ -46,6 +48,7 @@
  * @since 1.92
  */
 @Extension
+@Symbol("envinject")
```

Comment:
does this read better?
```suggestion
@Symbol("envInject")
```
(jcasc will resolve it case insensitively but it'll use the case on the symbol in the export)